packages/core/cli.js

```diff
@@ -40,16 +40,15 @@ if (!semver.gte(process.version, minimumNodeVersion)) {
 const listeners = process.listeners("warning");
 listeners.forEach(listener => process.removeListener("warning", listener));
 
-const inputStrings = process.argv.slice(2);
+let inputStrings = process.argv.slice(2);
 
-const userWantsGeneralHelp =
-  inputStrings.length === 0 ||
-  (inputStrings.length === 1 && ["help", "--help"].includes(inputStrings[0]));
+//check if user wants some help
+const helpNeeded = inputStrings.some(r => ["help", "--help"].indexOf(r) >= 0);
```

Comment:
you can simplify this a little bit to just use the `includes` method - using `indexOf` here feels a bit more confusing.

I would do 
```
inputStrings.some(inputString => ["help", "--help"].includes(inputString));
```
because you are really just trying to check whether "help" or "--help" is in the array, right? although this will also be triggered when the user is trying to access the help command as well

Reply:
I like it. thank you. Yes. help is help.

Reply:
cool. Thank you. I will change this. Yes help is help :-)